app/assets/stylesheets/application.css.scss

```diff
@@ -36,45 +37,6 @@ $image-path: '';
   }
 }
 
-.usa-label {
```

Comment:
i was able to get rid of a lot of our custom css as either things were fixed now, or i was able to use utility classes to achieve the same effect.